src/screens/ProfileScreen/elements/FollowInfoRow.js

```diff
@@ -69,13 +70,14 @@ const FollowInfoRow = ({
 let styles = StyleSheet.create({
   textTotal: {
     fontSize: 14,
-    fontWeight: '600',
-    color: COLORS.black000,
+    fontFamily: fonts.inter[600],
+    color: COLORS.white2,
     paddingRight: 4
   },
   textFollow: {
     fontSize: 14,
-    color: COLORS.black
+    fontFamily: fonts.inter[400],
+    color: COLORS.white2
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The font family and color of the text have been changed. Make sure these changes are consistent with the overall design system and do not affect readability or accessibility.

```diff
-    fontWeight: '600',
-    color: COLORS.black000,
+    fontFamily: fonts.inter[600],
+    color: COLORS.white2,

-    color: COLORS.black
+    fontFamily: fonts.inter[400],
+    color: COLORS.white2
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->